tests/test_repository.py

```diff
@@ -13,79 +13,20 @@
 from bci_tester.data import BCI_REPO_NAME
 from bci_tester.data import OS_SP_VERSION
 from bci_tester.data import OS_VERSION
-from bci_tester.data import REPOCLOSURE_CONTAINER
 
 
 _RM_ZYPPSERVICE = (
     "rm -v /usr/lib/zypp/plugins/services/container-suseconnect-zypp"
 )
 
-#: Packages that can be installed, but are reported as having dependency issues
-#: by :command:`dnf repoclosure`.
-#: This is caused by these packages having boolean requires on the kernel, which
-#: is not present in the SLE_BCI repository. We check that these packages can be
-#: installed in :py:func:`test_package_installation`.
-REPOCLOSURE_FALSE_POSITIVES = (
-    [
-        "multipath-tools",
-        "patterns-base-fips",
-        "patterns-base-minimal_base",
-        "podman",
-        "salt-minion",
-        "suse-module-tools",
-        "typelib-1_0-Gtk-3_0",
-    ]
-    + (
-        [
-            "qml-autoreqprov",
-            "typelib-1_0-Gtk-4_0",
-            "python311-aiohttp",
-            "python311-libcst",
-            "kernel-default-devel",
-        ]
-        if OS_SP_VERSION >= 4
-        else []
-    )
-    + (
-        [  # has a boolean dependency on NetworkManager
-            "jeos-firstboot",
-            # has a boolean dependency on the kernel
-            "ecryptfs-utils",
-        ]
-        if OS_SP_VERSION >= 6
-        else []
-    )
-    + (
-        ["open-vm-tools"]
-        if OS_SP_VERSION >= 5
-        and LOCALHOST.system_info.arch in ("aarch64", "x86_64")
-        else []
-    )
-    + (
-        ["open-vm-tools"]
-        if OS_SP_VERSION in (3, 4) and LOCALHOST.system_info.arch in ("x86_64")
-        else []
-    )
-)
-
-#: Packages that have broken dependencies by intention and should be excluded
-#: from the repoclosure checks
-KNOWN_BROKEN = [
-    #: aaa_base and kernel-rt_debug require 'distribution-release', which is
-    #: provided by `sles-release`.
-    #: However, `sles-release` is not in the repository, as we do not want
-    #: people to be able to build their own SLES from the SLE_BCI repo alone.
-    "aaa_base",
-]
-
 
 def get_package_list(con) -> List[str]:
     """This function returns all packages available from the ``SLE_BCI`` repository
     given a container connection.
 
     """
     package_list = con.check_output(
-        f"dnf list --available|grep -F '{BCI_REPO_NAME}' | cut -d' ' -f1",
+        f'zypper --xmlout se -r SLE_BCI | sed -n \'s/^<solvable.*name="\\([^"]*\\)".*$/\\1/p\'',
```

Comment:
I think in some other places I did the xml parsing in python to avoid messing with shell substitutions.

Reply:
Can do. Parsing xml with sed is not great, but then again zypper outputs pseudo-xml anyway...

Reply:
Done